### DISCUSSION TITLE
Unable to connect

### DISCUSSION THREAD

**Discussion by rdtennent** [jamulussoftware/jamulus#1878] [Technical Support & Help]
I've been running a private server on AWS Lightsail for several months and have now been trying to update the software to 3.8.0.  I first tried compiling the tar.gz but that failed (on Centos).  So i've set up a new instance running Ubuntu and have installed the .deb and activated the server.  But I'm unable to connect using either the dynamic IP reported by service jamulus-headless status or the static IP assigned by AWS.   Is there something else I need to do
(open ports, for instance)?  I'm not very familiar with Debian/Ubuntu conventions.

**Comment by gilgongo** [comment]
What's the output from `service jamulus-headless status`?

**Reply by rdtennent** [reply]
jamulus-headless.service - Jamulus headless server
     Loaded: loaded (/lib/systemd/system/jamulus-headless.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2021-06-15 13:30:10 UTC; 1h 42min ago
   Main PID: 397 (jamulus-headles)
      Tasks: 2 (limit: 555)
     Memory: 47.9M
     CGroup: /system.slice/jamulus-headless.service
             ââ397 /usr/bin/jamulus-headless -s -n

Jun 15 13:30:10 ip-172-26-11-205 systemd[1]: Started Jamulus headless server.
Jun 15 13:30:10 ip-172-26-11-205 jamulus[397]: - server mode chosen
Jun 15 13:30:10 ip-172-26-11-205 jamulus[397]: - no GUI mode chosen
Jun 15 13:30:11 ip-172-26-11-205 jamulus[397]: Using "172.26.11.205" as external IP.
Jun 15 13:30:11 ip-172-26-11-205 jamulus[397]: Recording state not initialised
Jun 15 13:30:11 ip-172-26-11-205 jamulus[397]:  *** Jamulus, Version 3.8.0
Jun 15 13:30:11 ip-172-26-11-205 jamulus[397]:  *** Internet Jam Session Software
Jun 15 13:30:11 ip-172-26-11-205 jamulus[397]:  *** Released under the GNU General Public License (GPL)

**Reply by rdtennent** [reply]
You can see the dynamic IP reported by the status output.  The static IP assigned by AWS is 3.98.103.219.  I can ssh to that IP address using the credentials set up by AWS so I know that IP is active. But a Jamulus client can't connect to either.

**Reply by henkdegroot** [reply]
Not sure if it helps, but I have setup Jamulus on a Google Cloud VM and just had to make sure I added the port forwarding for port 22124 in the firewall to allow the traffice to pass on from the external IP to the internal server IP.

**Reply by rdtennent** [reply]
I know how to open ports on Centos.  How do I do it on Ubuntu?

**Reply by gilgongo** [reply]
I would assume you'd do port forwarding outside of the server - in your AWS console?

Or have you got a local firewall running? That would be a bit unusual, but what does `sudo iptables -L` show?

**Reply by rdtennent** [reply]
I've used firewall-cmd to open open port 22124 and forward to 172.26.11.205.  iptables -L now has:  

Chain IN_public_allow (1 references)
target     prot opt source               destination         
ACCEPT     tcp  --  anywhere             anywhere             tcp dpt:ssh ctstate NEW,UNTRACKED
ACCEPT     udp  --  anywhere             anywhere             udp dpt:22124 ctstate NEW,UNTRACKED

Still no joy.

I guess I'll look for something in the AWS console.

**Reply by rdtennent** [reply]
Success!  There is a simple "add port" rule in the management console.  Thanks.
